### ISSUE TITLE
Tab triggers are missing aria-selected for accessibilty

### ISSUE THREAD

**Issue by GatienBouyer** [melt-ui/melt-ui#1231]
### Describe the bug

[Accessibility guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/) advise setting aria-selected on the active tab. Melt UI only has data-state on its tabs.

> The active tab element has the state [aria-selected](https://w3c.github.io/aria/#aria-selected) set to true and all other tab elements have it set to false.

Note:
- Tab seems to not be the only builder to miss aria-selected.
- Tab also misses aria-controls (on triggers/tab) and aria-labelledby (on content/tabpanel).

### Reproduction

1. Go to the [Melt UI Tab docs](https://www.melt-ui.com/docs/builders/tabs).
2. Inspect the button used to change tab (with role="tab").
3. Observe that aria-selected is not present.

### System Info

```bash
MeltUI v0.81.0
```

### Severity

annoyance
